Add package building scripts

caleridas opened the pull request:
Sample build command:

dpkg-buildpackage -us -uc -nc -b --host-arch arm64

https://auterion.atlassian.net/browse/DIS-456

caleridas committed:
Add package building scripts

Add minimal build scripts to build as ubuntu package. Set rules up such that we
can incrementally rebuild package cheaply.

QuentinVecchio commented on debian/copyright:
```diff
@@ -0,0 +1,38 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
```
Should we not use the LICENSE from mavlink repo instead?

caleridas replied:
Indeed, I failed to fix the license in this file -- fixed now.

QuentinVecchio reviewed (approved):
LGTM, thanks!

caleridas committed:
Set nice level for mavlink router

Bump nice level of mavlink router daemon in an attempt to make it more
responsive.

QuentinVecchio reviewed (approved):
LGTM
